File: XSNetwork/Classes/Client/Servers/Main/XSMainServer.m
//
//  TMDMainServer.m
//  TalkMed
//
//  Created by shun on 2017/8/24.
//  Copyright © 2017年 edoctor. All rights reserved.
//

#import "XSMainServer.h"

static NSString *releaseURL = @"";
static NSString *preReleaseURL = @"";
static NSString *devURL = @"";

@implementation XSMainServer

+ (void)setBaseURLWithRelease:(NSString *_Nullable)release dev:(NSString *_Nullable)dev preRelease:(NSString *_Nullable)preRelease {
    releaseURL = release ? release : @"";
    preReleaseURL = preRelease ? preRelease : @"";
    devURL = dev ? dev : @"";
}

@end

File: XSNetwork/Classes/Client/Servers/Main/XSMainServer.swift
//  XSMainServer.swift
//  XSNetwork

import Foundation

private var releaseURL: String = ""
private var preReleaseURL: String = ""
private var devURL: String = ""

@objcMembers
@objc public class XSMainServer: XSBaseServers {
    @objc public static func setBaseURLWithRelease(_ release: String?, dev: String?, preRelease: String?) {
        releaseURL = release ?? ""
        preReleaseURL = preRelease ?? ""
        devURL = dev ?? ""
    }
}
